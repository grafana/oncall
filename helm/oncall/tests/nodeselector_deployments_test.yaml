suite: test image and imagePullPolicy for deployments
templates:
  - celery/deployment.yaml
  - engine/deployment.yaml
  - engine/job-migrate.yaml
release:
  name: oncall
tests:
  - it: nodeSelector={} -> should set nodeSelector null
    asserts:
      - notExists:
          path: spec.template.spec.nodeSelector

  - it: nodeSelector -> should use custom nodeSelector
    set:
      migrate:
        nodeSelector:
          unittest: here
      engine:
        nodeSelector:
          unittest: here
      celery:
        nodeSelector:
          unittest: here
    asserts:
      - matchSnapshot:
          path: spec.template.spec.nodeSelector
