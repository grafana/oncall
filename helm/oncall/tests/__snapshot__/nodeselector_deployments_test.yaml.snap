nodeSelector -> should use custom nodeSelector:
  1: |
    unittest: here
  2: |
    unittest: here
  3: |
    unittest: here
